docs/docs/python-sdk/guides/python-typing.mdx

```diff
@@ -101,3 +101,51 @@ my_object = client.get(MyOwnObject, name__value="example")
 ```
 
 > if you don't have your own Python module, it's possible to use relative path by having the `procotols.py` in the same directory as your script/transform/generator
+
+## Generating Pydantic models from GraphQL queries
+
+When working with GraphQL queries, you can generate type-safe Pydantic models that correspond to your query return types. This provides excellent type safety and IDE support for your GraphQL operations.
+
+### Why use generated return types?
+
+Generated Pydantic models from GraphQL queries offer several important benefits:
+
+- **Type Safety**: Catch type errors at development time instead of runtime
```

Comment:
Suggested change:

```diff
- Catch type errors at development time instead of runtime
+ Catch type errors during development time instead of at runtime
```